index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+  <title>Home Page</title>
+</head>
+<body>
+  <header>
+    <div class="social">
+      <ul>
+        <li><a href="" class="mediaIcons"><i class="fa-brands fa-facebook-f"></i> <i class="fa-brands fa-twitter"></i></a></a></li>
+        <li><a href="">English</a></li>
+        <li><a href="">My Page</a></li>
+        <li><a href="">Logout</a></li>
+      </ul>
+    </div>
+      <nav class="mobile_nav">
+        <button class="hamburger">menu
+          <i class="fa-sharp fa-solid fa-xmark close whiteColor">
+          </i>
+          <i class="fa-solid fa-bars open"></i>
+        </button>
+        <a href="./index.html"><img src="./pictures/logoLion.png" class="logoLion" alt="Museum_logo" srcset=""></a>
+        <ul class="menu_list">
+          <li><a class="link_item whiteColor" href="./about.html">About</a></li>
+          <li><a class="link_item whiteColor" href="">Program</a></li>
+          <li><a class="link_item whiteColor" href="">Join</a></li>
+          <li><a class="link_item whiteColor" href="">Sponsor</a></li>
+          <li><a class="link_item whiteColor" href="">News</a></li>
+          <li><a class=" compaign" href="">NC Compaign</a></li>
+        </ul>
+      </nav>
+          <div class="banner">
+            <q class="salute">Hello! sharing world</q>
+            <h1 class="bigheader">NIGER CONNEXION<br> GLOBAL SUMMIT<br> 2023</h1>
+            <div class="paragraph">
+<p>
+  NIGER CONNEXION is an event agency whose niche is cultural entrepreneurship. Founded in 2009 and holder of a license
+  issued by the APEIC (Agence de Promotion des Entreprises et Industries Culturelles), composed of a young and dynamic
+  team always in search of performance and innovation.
+</p>
+            </div>
+            <div class="timeZone">
+              <h2 class="time">2015.10.15(THU) ~ 16(FRI)</h2>
+              <p class="place">@ Niger Connexion, Art Center Cultures and more</p>
+            </div>
+          </div>
+  </header>
+  <main class="container">
+    <div class="programs whiteColor" >
+      <h2 class="title">Main Program</h2>
+      <hr class="red_bar">
+      <div class="programs_cards" id="programs_cards">
```

Comment:
- [Optional] Great Job For updating previous reviewer suggestions, I think it would be if you use  `section` to hold a list of articles instead of `div` for better semantic reasons 👍👍